crates/primitives/src/chain/spec.rs

```diff
@@ -429,6 +429,16 @@ impl ChainSpec {
         &self.hardforks
     }
 
+    /// Get the fork id for the given hardfork.
+    pub fn hardfork_fork_id(&self, fork: Hardfork) -> ForkId {
+        self.fork_id(&self.fork(fork).satisfy())
+    }
```

Comment:
why is this not an option here? but on hardfork it is.

looks like this does not handler the ::Never case

Reply:
ah sorry about that - added.